src/content/blog.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Blog
+sort: 5
+redirect_to: https://medium.com/webpack
+contributors:
+  - pranshuchittora
+---
+
+
+<a href="https://medium.com/webpack">
```

Comment:
This is not only a link in sidebar itself this is a page you are creating with a link to blog?

Reply:
This is for the mobile sidebar, which is fine, but it should be a direct link, not a page itself.

Reply:
I do realize we already let vote "fix" slip. But we are in control and should change this on data structure level instead of creating placeholder md files in order to enrich the generated menu.

Reply:
My call is add vote and blog as part of array of the parsed pages for the menu on data level

Reply:
I have already mentioned in the PR description `Is there any way to link Blog directly to the medium page, instead of loading a separate page.`

IMO we must create a feature in Sidebar such that it accepts a `JSON` file `Sidebar.json` which specifies link name & it's respective URL.

``` JSON
{
  {
     "label" : "Vote",
     "url" : "/vote"
  },
  {
     "label" : "Blog",
     "url" : "https://medium.com/webpack"
  }
}
```

Reply:
You dont need a json file for two links. Just check out how navigation is implemented...
```
<Navigation
          pathname={location.pathname}
          toggleSidebar={this._toggleSidebar}
          links={[
            {
              content: 'Documentation',
              url: '/concepts',
              isActive: url => /^\/(api|concepts|configuration|guides|loaders|migrate|plugins)/.test(url),
              children: this._strip(sections.filter(item => item.name !== 'contribute'))
            },
            { content: 'Contribute', url: '/contribute' },
            { content: 'Vote', url: '/vote' },
            { content: 'Blog', url: 'https://medium.com/webpack' }
          ]}
        />
```

Reply:
mobile sidebar can be accepting all regular pages it accepts now plus two additional

Reply:
I tried appending the object in the array which is getting generated here :arrow_down: 
https://github.com/webpack/webpack.js.org/blob/956cfabc58ebb1a9f3ac8ae3a0296ff8531630cf/src/components/Site/Site.jsx#L151

Unfortunately, the SideBar is not producing the required result.

Reply:
needs a look then, therefore this PR is not ready